tee-worker/litentry/pallets/identity-management/src/identity_context.rs

```diff
@@ -94,7 +94,7 @@ pub fn get_eligible_identities<T: Config>(
 		.iter()
 		.filter_map(|item| {
 			if item.1.is_active() {
-				let mut networks = item.1.web3networks.to_vec();
+				let mut networks = item.0.default_web3networks();
 				// filter out identities whose web3networks are not supported by this specific `assertion`.
 				// We do it here before every request sending because:
 				// - it's a common step for all assertion buildings, for those assertions which only
```

Comment:
We can't use desired_web3networks as default all network because:
For example:
There is an account linked 2 identities:
Identity A: type is Substrate.
Identity B: type is Evm.
There is an assertion support 2 networkes: Litentry and Ethereum.
If request VC for this account and assertion:
The function get_eligible_identities use the assertion's supported web3networks as the value of desired_web3networks, so the value of desired_web3networks is [Ethereum, Litentry] for this case.
If use desired_web3networks as the result networks, the return value of this  function will be wrong:
[(Identity A, [Ethereum, Litentry]), (Identity B, [Ethereum, Litentry])]
Identity A is Substrate and Ethereum is not the Substrate network, Identity B is Evm and Litentry is not the Evm network, so the correct result should be:
[(Identity A, [Litentry]), (identity B, [Ethereum])]